src/views/RunPage.vue

```diff
@@ -19,11 +19,17 @@
             >
         </template>
         <template v-slot:default>
-            <LogOutput
-                v-for="captureArea in captureAreas"
-                :key="captureArea.getId()"
-                :captureAreaId="captureArea.getId()"
-            />
+            <v-alert v-if="isRunning" class="mb-4" type="warning"
+                >While capturing you are unable to switch changes sources or
+                adjust regex, please stop if you want to change those!</v-alert
+            >
```

Comment:
```suggestion
```
Unnecessary imo. I know you want to use these alert boxes, we'll find use for them at some point :D

Reply:
But the navigation is blocked as information to the user, but Ok 🌵